parser/src/main/java/io/moderne/jenkins/ingest/Merger.java

```diff
@@ -0,0 +1,78 @@
+package io.moderne.jenkins.ingest;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class Merger {
+
+    public static void main(String[] args) throws IOException {
+        mergeDatatables(Path.of(args[0]), Path.of(args[1]));
+    }
+
+    public static void mergeDatatables(Path original, Path newCsv) throws IOException {
+        Map<Key, CsvRow> newRows = parseNewCsv(newCsv);
+        updateReposCsv(original, newRows);
+    }
+
+    private static Map<Key, CsvRow> parseNewCsv(Path reposFile) throws IOException {
+        // header: scmHost,repoName,repoBranch,mavenTool,gradleTool,jdkTool,repoStyle,repoBuildAction,repoSkip,skipReason
+        List<String> reposLines = Files.readAllLines(reposFile);
+        return reposLines.stream()
+                .map(line -> line.split(",", -1))
+                .map(split -> new CsvRow(split[0].equals("github.com") ? "" : split[0],
+                        split[1], split[2], split[3], split[4], split[5], split[6], split[7], split[8], split[9]))
+                .collect(Collectors.toMap(row -> new Key(row.scmHost(), row.repoName(), row.repoBranch()), Function.identity()));
+    }
+
+    private static void updateReposCsv(Path reposFile, Map<Key, CsvRow> newRowsByKey) throws IOException {
+        // header: scmHost,repoName,repoBranch,mavenTool,gradleTool,jdkTool,repoStyle,repoBuildAction,repoSkip,skipReason
+        List<String> reposLines = Files.readAllLines(reposFile);
```

Comment:
A bit worried to load everything in memory.. Could we use a `BufferedReader` instead?

Reply:
I'll switch to `Files.lines(reposFile)` which returns a Stream; even better.